include/IECore/CubicBasis.h

```diff
@@ -44,6 +44,15 @@ IECORE_POP_DEFAULT_VISIBILITY
 namespace IECore
 {
 
+enum class StockCubicBasis
```

Comment:
@johnhaddon are you ok with this new enum and constructor and accessor? We're keen to get this out into production asap. I think @donboie will have a followup Gaffer PR using the stockBasis to display a basis-type in Gaffer's _Scene Inspector_ as well.

Reply:
It's a bit reminiscent of this stuff, which I was hoping would move to Cortex at some point :

https://github.com/GafferHQ/gaffer/blob/master/include/Gaffer/SplinePlug.h#L52
https://github.com/GafferHQ/gaffer/blob/master/include/Gaffer/SplinePlug.h#L62

That also has MonotoneCubic though, which isn't a basis, so I'm not sure how they would relate (maybe a generic Interpolation enum?). I don't have any concrete proposal here though - probably better to do what we need now than attempt a generalisation. Which just leaves me with bikeshedding...I wonder if `StandardCubicBasis::Bezier` sounds a bit better, or maybe just `StandardBasis::Bezier` or even `Basis::Bezier`? For what it's worth, I've kindof adopted `Standard<Woteva>` as a standard way of naming standard things in Gaffer...

Reply:
I vote for StandardCubicBasis as it matches the CubicBasis template class.

Reply:
Sounds good to me...

Reply:
Updated enum name in 6871ebf